scripts/update-terraform-modules.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+## Init temp dir
+tmp_dir=tmp
+mkdir -p $tmp_dir
+
+github_org=cloudposse
+github_api_limit=100
+github_api_pages=3
+
+modules=[]
+
+## Fetch repository names
+echo "Fetching repository names..."
+if [ ! -f $tmp_dir/modules_cache.txt ]; then
+    modules=$(
+        seq $github_api_pages | \
+        xargs -I "{}" curl -sSL "https://api.github.com/orgs/${github_org}/repos?per_page=${github_api_limit}&page={}" | \
+        jq -r '.[] | .name' | \
+        sort | grep terraform-*
+    )
+    echo "$modules" > $tmp_dir/modules_cache.txt
+else
+    modules=$(<$tmp_dir/modules_cache.txt)
+fi
+
+## Clone repositories
+echo "Cloning repositories..."
+mkdir -p $tmp_dir/projects/terraform
+for module in $modules
```

Comment:
This won't work. Modules are organized logically. 

![image](https://user-images.githubusercontent.com/52489/48283405-45b70780-e411-11e8-8ef4-72454b80241b.png)

Let's assume user creates the directories. Then we automatically generate docs for those directories.

https://github.com/cloudposse/docs/tree/master/content/terraform-modules/monitoring

Reply:
But how we can automate this process?
There isn't any meta-information about how to categorize modules.

Reply:
I don't think we need to add new modules automatically right now. We can do that manually.

Reply:
Ok.
So we need a strategy how to map terraform pages into user-created folders.

Reply:
Actually, what I meant to say is a user will run `touch terraform-aws-vpc.md` and then our script will go generate that file since it looks like `terraform-modules/.*/terraform-.*\.md`

I don't think we need a more elaborate mapping system.
One other consideration is can we generate documentation for multiple versions of the modules?  In that case, maybe we need to move from a file called `terraform-aws-vpc.md` to `terraform-aws-vpc/index.md`